sv/config.go

```diff
@@ -62,6 +63,7 @@ type VersioningConfig struct {
 // TagConfig tag preferences.
 type TagConfig struct {
 	Pattern string `yaml:"pattern"`
+	Filter  string `yaml:"filter"`
 }
 
```

Comment:
it would be great if you could add the both config attributes on `defaultConfig()` function, I know that don't make any difference since it's use the type default values, but it's easier to see all the default values defined explicit there..

Reply:
I added it.